### PR TITLE
CLOUDP-70063: mongocli ops-manager admin backup blockstore(s) describe

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-70063

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om admin backup blockstore describe blockstore -P ops
ID           URI                                     SSL     LOAD FACTOR
blockstore   mongodb://test1:test1@localhost:22222   false   1
```

```
./bin/mongocli om admin backup blockstore describe blockstore -P ops -o json
{
  "id": "blockstore",
  "uri": "mongodb://test1:test1@localhost:22222",
  "ssl": false,
  "assignmentEnabled": true,
  "encryptedCredentials": false,
  "loadFactor": 1
}
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting for https://github.com/mongodb/mongocli/pull/470

File: internal/cli/opsmanager/admin/backup/description.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package backup
+
+const (
+	Backup = "Manage backup administration for your project."
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/backup/description.go:17]: this is global so we can drop the project part

File: internal/cli/opsmanager/admin/description.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package admin
+
+const (
+	short = "Manage backup administration for your project."
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/description.go:17]: same let's drop project

**Review by gssbzn — CHANGES_REQUESTED** [review]
small copy change as the admin is global and not per project

File: internal/cli/opsmanager/admin/backup/blockstore/description.go [hunk 3]
```diff
@@ -14,6 +14,7 @@
 package blockstore
 
 const (
-	short = "Manage backup blockstores."
-	list  = "List backup blockstores."
+	short    = "Manage backup blockstores."
+	list     = "List backup blockstores."
+	describe = "Get a backup blockstore."
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/backup/blockstore/description.go:19]: maybe we should add the word configuration here, as in, "Manage backup blockstore configuration"

**Review by gssbzn — DISMISSED** [review]
LGTM one small comment but not blocking

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**